docs/serialization.md

```diff
@@ -43,3 +43,34 @@ user.toJSON(); /* {
 
 JSON.stringify(user)); // {"name":"John Something","birth":"1990-10-10T03:00:00.000Z","books":[{"name":"The name of the wind"},{"name":"Stonehenge"}]}
 ```
+
+### Use static toJSON to further modify serialized object:
```

Comment:
Maybe we could rename this section to `Customizing serialization output` and then explain it can be done using a static `toJSON` after the title, what do you think?